docker-sampledata.yml

```diff
@@ -0,0 +1,20 @@
+version: '3.7'
+services:
+  aleph:
+    image: mitlibraries/mario:aspace
```

Comment:
This and the one below are pinned to the `aspace` tag of Mario until that branch lands and then we should pin back to master.